js/git-read-issues/README.md

```diff
@@ -0,0 +1,24 @@
+# Git read issues
+
+This sequence reads all of the github issues for a given repositories
+
+it prepares the data in a format that will be used in [clickup-add-issues]() and send it to a topic.
+
+> ! All issues that has been read by this sequence will be labeled as read in a github
+
+## Setup
+
+Change data in `ghdata.json` to one that applies to you
+
+```json
+{
+    "owner": "owner", //Repository owner
+    "auth": "KEY", //API-KEY
```

Comment:
like in the Clickup Sequence, can we please move the key to be passed as an argument of the Sequence? 
thanks :)

Reply:
Thanks, this was already implemented. I've forgot to change readme file